bundles/org.openhab.binding.doorbird/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,224 @@
+			<channel id="doorbellHistoryImage" typeId="doorbellHistoryImage" />
+			<channel id="motionHistoryIndex" typeId="motionHistoryIndex" />
+			<channel id="motionHistoryTimestamp" typeId="motionHistoryTimestamp" />
+			<channel id="motionHistoryImage" typeId="motionHistoryImage" />
+			<channel id="getDoorbellMontage" typeId="getDoorbellMontage" />
+			<channel id="doorbellMontage" typeId="doorbellMontage" />
+			<channel id="getMotionMontage" typeId="getMotionMontage" />
+			<channel id="motionMontage" typeId="motionMontage" />
+			<channel id="sipHangup" typeId="sipHangup" />
+			<channel id="restart" typeId="restart" />
+		</channels>
+		<config-description-ref uri="thing-type:doorbird:config" />
+	</thing-type>
+
+	<channel-type id="doorbell">
+		<kind>trigger</kind>
+		<label>Doorbell</label>
+		<description>Doorbell pressed</description>
+        <event>
+            <options>
+                <option value="PRESSED">Pressed</option>
+                <option value="RELEASED">Released</option>
+            </options>
+        </event>
+	</channel-type>
+	<channel-type id="doorbellTimestamp">
+		<item-type>DateTime</item-type>
+		<label>Doorbell Timestamp</label>
+		<description>Time when doorbell was last pressed</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="doorbellImage">
+		<item-type>Image</item-type>
+		<label>Doorbell Pressed Image</label>
+		<description>Image when doorbell was last pressed</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="motion">
+        <kind>trigger</kind>
+		<label>Motion</label>
+		<description>Motion detected</description>
+        <event>
+            <options>
+                <option value="TRIGGERED">Triggered</option>
+                <option value="UNTRIGGERED">Untriggered</option>
+            </options>
+        </event>
+	</channel-type>
+	<channel-type id="motionTimestamp">
+		<item-type>DateTime</item-type>
+		<label>Motion Timestamp</label>
+		<description>Time when motion was last detected</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="motionImage">
+		<item-type>Image</item-type>
+		<label>Motion Detected Image</label>
+		<description>Image when motion was last detected</description>
+		<state readOnly="true"></state>
+	</channel-type>
+	<channel-type id="light">
+		<item-type>Switch</item-type>
+		<label>Light</label>
+		<description>Energize the light relay</description>
+		<state pattern="%s"></state>
+	</channel-type>
+	<channel-type id="openDoor1">
+		<item-type>Switch</item-type>
+		<label>Open Door 1</label>
+		<description>Energize opendoor / alarm output relay 1</description>
+		<state pattern="%s"></state>
+	</channel-type>
+	<channel-type id="openDoor2">
+		<item-type>Switch</item-type>
+		<label>Open Door 2</label>
+		<description>Energize opendoor / alarm output relay 2</description>
+		<state pattern="%s"></state>
+	</channel-type>
+	<channel-type id="getImage">
```

Comment:
What is the purpose of this channel? To update the ìmage` channel`? Why not force an update of the image channel by sending an `RefreshType.REFRESH` to it?